prisma/seeds/positions.ts

```diff
@@ -0,0 +1,78 @@
+import type { PrismaClient, Sport } from '@prisma/client'
+
+const seedPositions = (prisma: PrismaClient, sports: Sport[]) => {
+  const rugbyId = sports.find(sport => sport.key === 'rugby')!.id
+
+  const positions = [
+    {
+      key: 'prop',
+      numbers: [1, 3],
+      title: 'Prop',
+      category: 'forward',
+      sportId: rugbyId,
+    },
+    {
+      key: 'hooker',
+      numbers: [2],
+      title: 'Hooker',
+      category: 'forward',
+      sportId: rugbyId,
+    },
+    {
+      key: 'lock',
+      numbers: [4,5],
+      title: 'Lock',
+      category: 'forward',
+      sportId: rugbyId,
+    },
+    {
+      key: 'backrow',
+      numbers: [6,7,8],
+      title: 'Backrow',
+      category: 'forward',
+      sportId: rugbyId,
+    },
+    {
+      key: 'scrumhalf',
+      numbers: [9],
+      title: 'Scrumhalf', // 'scrumhalf' or 'halfback'
+      category: 'back',
+      sportId: rugbyId,
+    },
+    {
+      key: 'flyhalf',
+      numbers: [10],
+      title: 'Flyhalf',
+      category: 'back',
+      sportId: rugbyId,
+    },
+    {
+      key: 'centre',
+      numbers: [12,13],
+      title: 'Center',
+      category: 'back',
+      sportId: rugbyId,
+    },
+    {
+      key: 'outside_back',
+      numbers: [11,14,15],
+      title: 'Outside Back',
+      category: 'back',
+      sportId: rugbyId,
+    },
+  ]
+
+  const records = positions.map(async position => (
+    await prisma.position.upsert({
+      where: { 
+        key: { key: position.key }
```

Comment:
Issue is here

![Screenshot 2023-11-05 at 21 29 05](https://github.com/dcarg/teamsheet/assets/39669538/a4844585-dfae-480e-bfe9-dc29f22f908d)

Reply:
you have an extra nested object here, should just be
```
where: { 
  key: position.key,
},
```